lte/gateway/python/magma/mobilityd/tests/test_ip_allocator_dhcp.py

```diff
@@ -0,0 +1,310 @@
+"""
```

Comment:
In bazel this is marked as a sudo test - should the test cases be annotated with something like `@unittest.skipIf(os.getuid(), reason="needs root user")`?

Reply:
I think we don't actually need sudo for these tests, as the script that would require it is mocked. I removed the flag in bazel.